pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -571,6 +573,25 @@ type HTTPConfig struct {
 	FollowRedirects *bool `json:"followRedirects,omitempty"`
 }
 
+// WebexConfig configures notification via Cisco Webex
+// See https://prometheus.io/docs/alerting/latest/configuration/#webex_config
+type WebexConfig struct {
+	// The Webex Teams API URL i.e. https://webexapis.com/v1/messages
+	// +optional
+	APIURL *string `json:"apiURL,omitempty"`
```

Comment:
Let's create a URL type for proper validation
```suggestion
	APIURL *URL `json:"apiURL,omitempty"`
```

then

````
// +kubebuilder:validation:Pattern=`^https?://.+$`
type URL string
````

Reply:
Should I also replace other URL-kind fields as with the new URL type (maybe in a separate PR)?

Reply:
> Does a dedicated URL type offer any other advantages, which I'm not aware of?

yes, `kubebuilder:validation:Pattern` is applied by the Kubernetes API without requiring any validating webhook configuration.